src/main/java/pizzashop/controller/OrdersGUIController.java

```diff
@@ -145,16 +149,15 @@ private void initData(){
                     .collect(Collectors.toList());
             setTotalAmount(orderPaymentList.stream().mapToDouble(e->e.doubleValue()).sum());
             orderStatus.setText("Total amount: " + getTotalAmount());
-            System.out.println("--------------------------");
+            System.out.println(ix);
```

Comment:
ce e ix de il pui in locul liniutelor ce erau inainte? idem pt toate locurile unde ai facut asta

Reply:
e o variabila ,imi zicea sonaru ca in apar liniutele in prea multe locuri,si sa le inlocuiesc cu o variabila. am denumit-o sugestiv ix :))

Reply:
ok